src/ephemeron/dune

```diff
@@ -0,0 +1,41 @@
+;; Linearizability tests of the stdlib Ephemeron module
+
+;; this prevents the tests from running on a default build
+(alias
+ (name default)
+ (deps lin_tests_dsl.exe))
```

Comment:
`stm_test.exe` is missing here as a dependency on "`default`".
This ensures that it is compiled when `dune build` is called.

Reply:
Done in f277d63